src/ipmi.c

```diff
@@ -441,7 +647,7 @@ static void entity_sensor_update_handler(
  */
 static void domain_entity_update_handler(
     enum ipmi_update_e op, ipmi_domain_t __attribute__((unused)) * domain,
-    ipmi_entity_t *entity, void __attribute__((unused)) * user_data) {
+    ipmi_entity_t * entity, void __attribute__((unused)) * user_data) {
```

Comment:
This looks like something *clang-format* should take care of. Please make sure to run it when preparing commits.